tutorials/index.md

```diff
@@ -27,3 +29,18 @@ Python:
 -   [Case Study: Analyzing Genomic Data with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_genomic_data.ipynb){:target="_blank"}
 
 -   [Case Study: Drug Discovery with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/Drug_Discovery_With_Data_Commons.ipynb){:target="_blank"}
+
+
+### India
+-   [Analysis of Mean Daily Wages of Men and Women of India](https://colab.research.google.com/drive/1oL0v6ZB4InHnMYo3bTQcSs4ZIHz6XIQ7#scrollTo=GE0b7K-Tiq8p){:target="_blank"}
+
+-   [State-wise Infant Heath Analysis](https://colab.research.google.com/drive/1hiDoxB46QR8ZoV5yEm4WIFfNF8E-jDbs#scrollTo=BG_H4WFiKj5B){:target="_blank"}
+
+-   [Analyzing the Relation between Fertility Rate along with Life Expectancy and Infant Mortality Rate](https://colab.research.google.com/drive/1rYjWjeZFU6b2lvaNvs9hKTfx75d4_r7i){:target="_blank"}
+
+-   [Analyzing Earthquake Magnitude Levels of India](https://colab.research.google.com/drive/15EPTLU3bKyf95av87xHUMKzXL_WehtNA){:target="_blank"}
+
+-   [Analyzing the relation between Fertility Rate and the Number of Vasectomies and Tubectomies](https://colab.research.google.com/drive/1_heEp-E6dEMJoJmlbKu5avb2R2WdJnGI){:target="_blank"}
```

Comment:
See my comments on a copy of this colab here:  https://colab.research.google.com/drive/19xUhJmhFuOAsA-nuULC_Xc3v4j-RW9FX?usp=sharing